File: Python/mathematical calculator.py
# take two input numbers
number1 = input("Insert first number : ")
number2 = input("Insert second number : ")

operator = input("Insert operator (+ or - or * or /)")

if operator == '+':
    total = number1 + number2
elif operator == '-':
    total = number1 - number2
elif operator == '*':
    total = number2 * number1
elif operator == '/':
    total = number1 / number2
else:
    total = "invalid operator"


# printing output
print(total)

File: JavaScript/Mathematical Calculator.js
// program for a mathematical calculator


const number1 = parseFloat(prompt('Enter first number: '));
const number2 = parseFloat(prompt('Enter second number: '));

const operator = prompt('Enter operator ( either +, -, * or / ): ');

let result;

// using if...else if... else
if (operator == '+') {
    result = number1 + number2;
}
else if (operator == '-') {
    result = number1 - number2;
}
else if (operator == '*') {
    result = number1 * number2;
}
else if (operator == '/'){
    result = number1 / number2;
}

// display the result
console.log(`${number1} ${operator} ${number2} = ${result}`);